testsuite/MDAnalysisTests/data/mol2/Molecule.mol2

```diff
@@ -7,7 +7,7 @@ USER_CHARGES
 
 @<TRIPOS>ATOM
   1 N1       6.8420     9.9900    22.7430 N.am  1 Q101  -0.8960 
-  2 S1       8.1400     9.2310    23.3330 S.o2  1 Q101   1.3220 

```

Comment:
This might be the only case for inconsistency, since all other strings after `.` are lowercase in the standard. My preference would be to support `S.o2` and `S.o` only as special case (probably coming from https://github.com/openbabel/openbabel/issues/599 ?) and eventually switch to `lower()` if other elements are affected by similar problems, but happy to be convinced otherwise.

Reply:
Yeah just supporting those special cases sounds good to me.

Reply:
P.S. You'll probably want to revert the changes to these files, we might as well keep them as-is (as a record of how they were originally created). I'd also recommend a test that reads in a file with all the supported atom types + at least one unexpected one that way we can be sure that everything works as expected.